seahorn/trusty/user/base/lib/tipc/proof/tipc_connect/Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
this file should not have been included. 
Same applies to all files in `Testing/` directory